ncm-network/src/main/pan/components/network/core-schema.pan

```diff
@@ -77,18 +77,28 @@ type structure_rule = {
     "table" ? network_valid_routing_table
     @{priority, The priority of the rule over the others. Required by Network Manager when setting routing rules.}
     "priority" ? long(0..0xffffffff)
+    @{action used by nmstate module}
+    "action" ? choice('unreachable', 'blackhole', 'prohibit')
```

Comment:
Would it make sense to sort the choices alphabetically? (b, p, u) ?

If this is exclusive to nmstate and ignored by other methods, use a nmstate- prefix (i.e. nmstate-action) for clarity?

Reply:
make sense.